scripts/run-example.sh

```diff
@@ -87,7 +87,11 @@ then
   ${HELM} upgrade --install vault hashicorp/vault --namespace vault --values "${SAMPLE_DIR}/helm-vault-values.yaml" --wait
   ${KUBECTL} exec vault-0 -n vault -- vault secrets enable transit || true
   echo -e "${GREEN}HashiCorp Vault installed.${NOCOLOR}"
-  VAULT_UI_URL=$(${MINIKUBE} service vault -n vault --url 2>/dev/null | head -1)
+
+  VAULT_UI_URL=$(head -1 <(${MINIKUBE} service vault -n vault --url 2>/dev/null &))
+  #  minikube_service_pid=$!
```

Comment:
Do we need the commented part?

Reply:
The issue is the the minikube service process will accumulate in the background.  We have no way to know when to shut them down. Suggest we echo the pid and tell the user to kill the process once they are done with the example.

Reply:
@k-wall my original thinking had been to get the URL and then terminate the background process and then tell users to forward ports as we did previously that way its consistent regardless of the arch/driver.

Either way I still need a reliable way of getting the PID of the minikube process and not the bash subshell/head command.